content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Enable the Evaluate Artifacts Stage Plugin
+description: >
+  Enable the Evaluate Artifacts Stage plugin to add a stage to Armory Enterprise that makes it easy to evaluate SpEL expressions inside of Spinnaker™ artifacts.
+aliases:
+  - /docs/plugin-guide/plugin-evaluate-artifacts/
+---
+
+![Proprietary](/images/proprietary.svg)
```

Comment:
Does this mean the plugin only works with Armory Enterprise?

Reply:
It means it's a proprietary plugin and is not open source

Reply:
Ok. Stephen's blog post says it works with AE and Spinnaker, so seeing the Proprietary graphic made me think it only works with AE.